client/src/services/AuthService.js

```diff
@@ -0,0 +1,42 @@
+module.exports = function(app) {
+
+app.service('AuthService', function AuthService($q, $http) {
```

Comment:
Try not to use injection notation without explicit declaration of injected items. Because using of uglification will break injected params names and injector will not find requested services and controllers. I suggest always to use array-notation or $inject property of constructor function.
```
app.service('AuthService', ['$q', '$http', function AuthService($q, $http) {...} ]);
````